include/externals/cub/thread/thread_store.cuh

```diff
@@ -145,10 +145,10 @@ template <int MAX>
 struct IterateThreadStore<MAX, MAX>
 {
     template <CacheStoreModifier MODIFIER, typename T>
-    static __device__ __forceinline__ void Store(T * /*ptr*/, T * /*vals*/) {}
+    static __device__ __forceinline__ void Store(T *ptr, T *vals) {}
```

Comment:
The changes on lines 148 and 151 were accidental by me.  It's the change on 351 is what counts.  Can you revert the changes on lines 148 and 151 with your next commit @weinbe2

Reply:
Done locally!